api/src/auth/drivers/saml.ts

```diff
@@ -88,15 +86,18 @@ export class SAMLAuthDriver extends LocalAuthDriver {
 
 		// Run hook so the end user has the chance to augment the
 		// user that is about to be created
+		const schema = await getSchema();
```

Comment:
A very tiny thing but lets make sure these comments stay with the line they refer to 😄 
```suggestion
		const schema = await getSchema();
		
		// Run hook so the end user has the chance to augment the
		// user that is about to be created
```

Reply:
Ah yeah you are right sorry for that mistake 😅